master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
```

Comment:
Ok, but then you go to this page:
http://docs.buildbot.net/latest/manual/cfg-www.html#web-authentication

And it says the class is "buildbot.www.auth.UserPasswordAuth", so that's what I was basing my patch on... As I was reading the docs, the "util" form never was explained and I assumed the text snippets (that I put in this patch) were all wrong.